docs/source/actions.rst

```diff
@@ -0,0 +1,38 @@
+Actions
+=======
+
+Actions are supported in the end of the alert pipeline currently.
```

Comment:
change "in" to "at"

Reply:
I like that better, added in https://github.com/mozilla/MozDef/pull/1235/commits/9d45ad5415db037ad647d01a0f4049f6bb0a6330